components/brave_rewards/resources/BUILD.gn

```diff
@@ -8,9 +8,11 @@ pack_web_resources("page_generated_resources") {
   resource_name = "brave_rewards_page"
   output_dir = "$root_gen_dir/brave/components/brave_rewards/resources"
   # need webpack and GRD generation first
-  deps = [
-    "page"
-  ]
+  if (is_android) {
```

Comment:
do we need new name?

Reply:
Do you mean `android_page`? It's just the directory where the Android-specific pages are located. They were unfortunately different enough that I had to put them in a separate location.